ksp/src/main/kotlin/fp/serrano/kopykat/utils/KspExtensions.kt

```diff
@@ -2,12 +2,21 @@ package fp.serrano.kopykat.utils
 
 import com.google.devtools.ksp.symbol.KSClassDeclaration
 import com.google.devtools.ksp.symbol.KSDeclaration
+import com.google.devtools.ksp.symbol.KSDeclarationContainer
 import com.google.devtools.ksp.symbol.KSPropertyDeclaration
 
-internal val KSDeclaration.name get() = simpleName.asString()
+internal val KSDeclaration.baseName get() = simpleName.asString()
+
+internal val KSDeclaration.qfName get() = qualifiedName?.asString() ?: simpleName.asString()
 
 internal val KSClassDeclaration.sealedTypes get() = getSealedSubclasses()
 
+internal fun KSDeclarationContainer.allNestedDeclarations(): Sequence<KSDeclaration> =
+internal fun KSDeclarationContainer.allNestedDeclarations(): Sequence<KSDeclaration> =
```

Comment:
Sorry @serras My suggestion destroyed the build :grimacing:

Reply:
![Curse You!](https://media.giphy.com/media/MwtjauC5lamQM/giphy.gif)